python/rmm/_lib/memory_resource.pxd

```diff
@@ -112,3 +112,7 @@ cdef class HybridMemoryResource(MemoryResource):
 
 cdef class LoggingResourceAdaptor(MemoryResource):
     cpdef flush(self)
+
+cpdef get_default_resource_type()
+
+cpdef is_initialized()
```

Comment:
Is this needed?

Reply:
Answered the same question above. So marking as resolved. Let's discuss above.